pyproject.toml

```diff
@@ -95,90 +113,88 @@ zip-safe = false
+    "pars/hap_pars/mvm_parameters/acs/hrc/*",
+    "pars/hap_pars/mvm_parameters/acs/sbc/*",
+    "pars/hap_pars/mvm_parameters/acs/wfc/*",
+    "pars/hap_pars/mvm_parameters/any/*",
+    "pars/hap_pars/mvm_parameters/wfc3/ir/*",
+    "pars/hap_pars/mvm_parameters/wfc3/uvis/*",
+    "pars/hap_pars/mvm_parameters/wfpc2/wf/*",
+    "pars/hap_pars/mvm_parameters/wfpc2/pc/*",
+    "pars/hap_pars/svm_parameters/*",
+    "pars/hap_pars/svm_parameters/acs/hrc/*",
+    "pars/hap_pars/svm_parameters/acs/sbc/*",
+    "pars/hap_pars/svm_parameters/acs/wfc/*",
+    "pars/hap_pars/svm_parameters/any/*",
+    "pars/hap_pars/svm_parameters/wfc3/ir/*",
+    "pars/hap_pars/svm_parameters/wfc3/uvis/*",
+    "pars/hap_pars/svm_parameters/wfpc2/wf/*",
+    "pars/hap_pars/svm_parameters/wfpc2/pc/*",
+    "pars/psfs/*",
+    "pars/psfs/acs/hrc/*",
+    "pars/psfs/acs/sbc/*",
+    "pars/psfs/acs/wfc/*",
+    "pars/psfs/wfc3/ir/*",
+    "pars/psfs/wfc3/uvis/*",
+    "*.help",
+    "htmlhelp/*",
+    "htmlhelp/_*/*",
+    "htmlhelp/_*/*/*",
 ]
 
 [tool.build_sphinx]
-builder = 'html'
-configuration = 'docs/source/conf.py'
+builder = "html"
+configuration = "docs/source/conf.py"
 fail_on_warning = false
 
-
 [tool.pytest.ini_options]
 minversion = 5
 norecursedirs = [
-    '.eggs',
-    'build',
-    '.tox',
-    'doc/build',
-    'doc/exts'
+    ".eggs",
+    "build",
+    ".tox",
+    "doc/build",
+    "doc/exts",
 ]
-junit_family = 'xunit2'
-inputs_root = 'drizzlepac'
-results_root = 'drizzlepac-results'
+junit_family = "xunit2"
+inputs_root = "drizzlepac"
+results_root = "drizzlepac-results"
 
 [tool.ruff]
 extend-exclude = [
-    '.git',
-    '__pycache__',
-    'build',
-    'dist',
-    '.eggs',
-    '.tox',
+    ".git",
+    "__pycache__",
+    "build",
+    "dist",
+    ".eggs",
+    ".tox",
 ]
 ignore = [
-    'E401', # Multiple imports not allowed on single line
-    'E402', # Module level import not at top of file
-    'E501', # Line too long
-    'E701', # Multiple-statements on one line (colon) NEEDS TO BE FIXED
-    'E711', # Comparison to `None` should be `cond is None`
-    'E712', # Comparison to `True` should be `cond is True`
-    'E713', # Test for membership should be `not in`
-    'E714', # Test for object identity should be `is not`
-    'E721', # Do not compare types, use `isinstance()`
-    'E722', # Do not use bare `except`
-    'E741', # Ambiguous variable name
-    'F401', # imported but unused
-    'F521', # '...'.format(...) has invalid format string: Single '}' encountered in format string
-    'F523', # '...'.format(...) has unused arguments at position(s): 0, 1
-    'F541', # f-string without any placeholders
-    'F632', # Use `==` and `!=` to compare constant literals
-    'F633', # Use of `>>` is invalid with `print` function
-    'F811', # Redefinition of unused
-    'F821', # Undefined name
-    'F822', # Undefined name `getHelpAsString` in `__all__`
-    'F841', # Local variable is assigned to but never used
+    "E401",
```

Comment:
Just a question here.  Is it against the new rules to have the commentary associated with the error codes to ignore.  The commentary is rather helpful.  Thanks.

Reply:
oh, sorry! That must have been the formatter; I'll add those comments back in